### ISSUE TITLE
SSL TLV should be hex 0x20

### ISSUE THREAD

**Issue by nazar554** [misalcedo/ppp#21]
There is a typo in the definition of `PP2_TYPE_SSL`, it's `20` in decimal and it should be in hex.

https://github.com/misalcedo/ppp/blob/2eebc457e290afa5c30afe2c136bd24e4f2cb7d9/src/v2/model.rs#L132

See [proxy-protocol.txt](https://www.haproxy.org/download/2.6/doc/proxy-protocol.txt)

```c
#define PP2_TYPE_SSL            0x20
#define PP2_SUBTYPE_SSL_VERSION 0x21
#define PP2_SUBTYPE_SSL_CN      0x22
```

**Comment by misalcedo** [comment]
Fixed by https://github.com/misalcedo/ppp/pull/20/files

**Comment by misalcedo** [comment]
Fixed version 2.2.0 should be in crates.io now: https://github.com/misalcedo/ppp/runs/7163932340?check_suite_focus=true
